modules/govuk_rbenv/manifests/all.pp

```diff
@@ -29,10 +29,13 @@
     '2.6.3',
     '2.6.5',
     '2.6.6',
+    '2.6.9',
     '2.7.0',
     '2.7.1',
     '2.7.2',
     '2.7.3',
+    '2.7.5',
+    '3.0.3',
```

Comment:
This [now needs](https://github.com/alphagov/packager/pull/196#issuecomment-1017807379) the 3.1.0 version too.